test/subplot/polarcases.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Tests should be grouped by topic. These tests are not for testing `subplot` and should be moved to other directories, e.g. projection.